Example/Example.xcodeproj/project.pbxproj

```diff
@@ -80,8 +74,6 @@
 		1C9404EC1B31D057001BF122 = {
 			isa = PBXGroup;
 			children = (
-				1C4B1A461B37F491008720C6 /* GaugeKit.framework */,
-				1C1E3C561B34CFE2005F495B /* GaugeKit.framework */,
```

Comment:
Ok I was getting an error when trying to run the Example because xcode was looking for the .framework file relative to the home dir on your machine and I had to delete it to get it to work, but I'll checkout changes on that file.